src/Summoner/Template/Cabal.hs

```diff
@@ -79,8 +79,8 @@ cabalFile Settings{..} = File (toString settingsRepo ++ ".cabal") cabalFileConte
           exposed-modules:     $libModuleName
                                $preludeMod
 
-          build-depends:       $settingsBase
-                             $commaPreludeLibrary
+          build-depends:       $settingsBaseType
+                               $commaPreludeLibrary
```

Comment:
This two spaces shouldn't be added. Basically, this line contains the comma in it, so after rendering it looks like this:
```
          build-depends:       base
                             , relude
```

But, if you add this two spaces you will get:
```
          build-depends:       base
                               , relude
```

which is not correct.